lib/app.dart

```diff
@@ -204,14 +217,12 @@ class AppState extends State<App> with WidgetsBindingObserver {
           // Wrap in popscope
           return MaterialPageRoute(
             builder: (BuildContext context) {
-              return WillPopScope(
-                onWillPop: () async {
-                  // On the pinscreen, background instead of pop
+              return PopScope(
+                canPop: settings.name != PinScreen.routeName,
+                onPopInvokedWithResult: (didPop, popResult) {
+                  // On the pin screen, background instead of pop
                   if (settings.name == PinScreen.routeName) {
```

Comment:
`didPop` can be reused here, right? Something like this:

```suggestion
                  if (!didPop) {
```

Reply:
Yes good spot. I used to have it that way, but it caused issues on Android.. Maybe I'll take a closer look.

Reply:
It's not that important to do extensive research. If you find a fix for it, it would be nice though.

Reply:
Apparently it's not an issue (anymore), so I made it `!didPop` like you suggested  :-) 
Thanks for the review!